### PR TITLE
Show the carbon saved by using Passim

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: src/passim-server.c [hunk 1]
```diff
@@ -177,11 +251,7 @@ passim_server_eventlog(PassimServer *self, const gchar *type, const gchar *value
 	g_autoptr(GFileOutputStream) ostream = NULL;
 
 	/* create logdir */
-	if (logs_directory == NULL) {
-		path = g_build_filename(PACKAGE_LOCALSTATEDIR, "log", PACKAGE_NAME, NULL);
-	} else {
-		path = g_strdup(logs_directory);
-	}
+	path = passim_server_get_logdir();
```
Comment (kalvdans) [inline src/passim-server.c:254]: `path` is never free'd

**Review by kalvdans — COMMENTED** [review]
Better make the server expose `total_shared` and leave the conversion to other units to each client.

File: src/passim-server.c (hunk 1 above)
Comment (hughsie) [reply src/passim-server.c:254]: Ohh it's declared as `g_autofree gchar *path = NULL` which means it's free'd when it goes out of scope. RII for the win.

Comment (kalvdans) [reply src/passim-server.c:259]: I see, good! :)

**Pull request merged**